draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2237,18 +2237,18 @@ relies on a random oracle. Applications using nonuniform encodings SHOULD carefu
 analyze the security implications of using such an encoding. When the required
 encoding is not clear, applications SHOULD use a random oracle.
 
-The below table lists the curves for which suites are defined and
-the subsection that gives the corresponding parameters.
-
-| E                         | Section              |
-|---------------------------|----------------------|
-| NIST P-256                | {{suites-p256}}      |
-| NIST P-384                | {{suites-p384}}      |
-| NIST P-521                | {{suites-p521}}      |
-| curve25519 / edwards25519 | {{suites-25519}}     |
-| curve448 / edwards448     | {{suites-448}}       |
-| secp256k1                 | {{suites-secp256k1}} |
-| BLS12-381                 | {{suites-bls12381}}  |
+The below table lists suites RECOMMENDED for some elliptic curves. The
+corresponding parameters are given in the following subsections.
+
+| E                       | Section              | Suites |
+|-------------------------|----------------------|--------|
+| NIST P-256              | {{suites-p256}}      | P256\_XMD:SHA-256\_SSWU\_RO\_ P256\_XMD:SHA-256\_SSWU\_NU\_ |
+| NIST P-384              | {{suites-p384}}      | P384\_XMD:SHA-512\_SSWU\_RO\_ P384\_XMD:SHA-512\_SSWU\_NU\_ |
+| NIST P-521              | {{suites-p521}}      | P521\_XMD:SHA-512\_SSWU\_RO\_ P521\_XMD:SHA-512\_SSWU\_NU\_ |
+| curve25519 edwards25519 | {{suites-25519}}     | curve25519\_XMD:SHA-512\_ELL2\_RO\_ curve25519\_XMD:SHA-512\_ELL2\_NU\_ edwards25519\_XMD:SHA-512\_ELL2\_RO\_ edwards25519\_XMD:SHA-512\_ELL2\_NU\_ |
+| curve448 edwards448     | {{suites-448}}       | curve448\_XMD:SHA-512\_ELL2\_RO\_ curve448\_XMD:SHA-512\_ELL2\_NU\_ edwards448\_XMD:SHA-512\_ELL2\_RO\_ edwards448\_XMD:SHA-512\_ELL2\_NU\_ |
```

Comment:
It might be nice to break these up into separate lines in the table: one line for curve25519 that lists relevant suites, one for edwards25519, etc.